songwhip/songwhip.js

```diff
@@ -7,18 +7,25 @@
 
 (function Songwhip() {
 	const SW_TEXT = 'Songwhip!';
-	const SW_URL = 'https://songwhip.com';
+	const SW_URL = 'https://songwhip.com/api';
+	const SW_BASE_URL = 'https://songwhip.com';
+	const SW_CLASS = 'songwhip-css';
+	const SW_PAR_CLASS = 'sw-par';
+	const COUNTRY = 'US';
+	const ERROR_TITLE = 'Error';
+	const ERROR_CONTENT = 'Failed fetching Songwhip :(';
+	const STYLE_CONTENT = `
+		.${SW_PAR_CLASS} {
+			clear: right !important;
+			margin-bottom: 1em !important;
+		}
+	`;
```

Comment:
I don't feel comfortable having these here, what is the rationale?

Reply:
I don't know if it's weird but usually, if I have relatively large strings I tend to put the at the beginning of my code as constants to make the methods and other parts of the code cleaner, dk if it's correct through 😅.

Reply:
Hah, neither do I 🤣. It's not like this will be some corporate-level codebase, so we'll just keep that as is; only was curious why you went with that.